static/js/Hello.jsx

```diff
@@ -17,9 +17,8 @@ export default class Hello extends React.Component {
     }
 
     getPythonHello() {
-        $.get(window.location.href + "hello", (data) => {
-            console.log(data);
-            this.personaliseGreeting(data);
+        $.get(window.location.href + "hello", () => {
```

Comment:
- You should be using `window.location.origin` here instead of `window.location.href` `href` will be different depending on your current location with relation to the website routing. But `origin` will always be the same. **Example from this page**:

![image](https://user-images.githubusercontent.com/26286907/79273623-f4376580-7eab-11ea-86ee-8342af23e7cf.png)

- Since we're using `GET` now we should pass the values through the url. **Example**:
```javascript
const language = 'en-us'
let text = 'something to say'
const url = `${origin}/${language}/${text}`

$.get(url, () => { 
...
```

Reply:
It makes sense ! I need to start working on the frontend, that was just a template i had from another app and tried to use it to trigger the server.
But this also applies if i take the route you sugested, the Fetch API instead of jquery ?

Reply:
Definitely, Fetch API all the way!

Reply:
I will start reading the docs and work on it when i start working on the frontend.